Exercise_1.java

```diff
@@ -1,46 +1,89 @@
-class Stack { 
+import java.util.*;
+
+class StackArr { 
     //Please read sample.java file before starting.
-  //Kindly include Time and Space complexity at top of each file
+    //Kindly include Time and Space complexity at top of each file
+    //Time complexity is O(1) as no loops are involved
+    //Space complexity is O[1] as well 
+
     static final int MAX = 1000; 

```

Comment:
Try to use linked list or some other data structure to implement stack, then there won't be any max size variable